tests/create_LWE.py

```diff
@@ -0,0 +1,322 @@
+        unit = configure_tr(component_system, **kwargs)
+    elif unit_type == 'lumpedratemodelwithoutpores':
+        unit = configure_lrm(component_system, **kwargs)
+    elif unit_type == 'lumpedratemodelwithpores':
+        unit = configure_lrmp(component_system, **kwargs)
+    elif unit_type == 'mct':
+        unit = configure_mct(component_system, **kwargs)
+    else:
+        raise ValueError(f'Unknown unit operation type {unit_type}')
+
+    setup_flow_sheet(flow_sheet, unit, component_system)
+
+    process = Process(flow_sheet, 'process')
+    process.cycle_time = 120 * 60
+
+    c1_lwe = [[50.0], [0.0], [[100.0, 0.2]]]
+    cx_lwe = [[1.0], [0.0], [0.0]]
+
+    process.add_event('load', 'flow_sheet.inlet.c',
+                      c1_lwe[0] + cx_lwe[0] * (n_comp - 1), 0)
+    process.add_event('wash', 'flow_sheet.inlet.c',
+                      c1_lwe[1] + cx_lwe[1] * (n_comp - 1), 10)
+    process.add_event('elute', 'flow_sheet.inlet.c',
+                      c1_lwe[2] + cx_lwe[2] * (n_comp - 1), 90)
+
+    return process
+
+
+def run_simulation(process: Process, cadet_install_path: str = None) -> NoReturn:
+    process_simulator = Cadet(install_path=cadet_install_path)
+    simulation_results = process_simulator.simulate(process)
+
+    cadet_version, branch_name = get_cadet_version(process_simulator.cadet_path)
+
+    if cadet_version < '5.0.0' and branch_name == 'GITDIR-NOTFOUND branch':
+        warnings.warn(
+            f"Your current CADET-core release version {cadet_version} may not support all unit operations in this CADET-process version. "
+            f"Please update to the latest CADET-core version from https://github.com/modsim/CADET for full compatibility.",
+            UserWarning
+        )
+
+    if simulation_results.exit_flag == 0:
+        print(
+            f"CADET-Process LWE test simulation for unit {process.flow_sheet.unit_names[1]} completed successfully")
+    else:
+        raise CADETProcessError(
+            f"CADET-Process LWE test simulation failed with {simulation_results.exit_message}"
+        )
+
+
+def get_cadet_version(cadet_install_path: str) -> tuple[str, str]:
+    try:
+        version_output = subprocess.run(
+            [cadet_install_path, '--version'],
+            check=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True
+        ).stdout.strip()
+
+        version_match = re.search(
+            r'cadet-cli version ([\d.]+) \(([^)]+)\)', version_output)
+
+        if version_match:
+            cadet_version = version_match.group(1)
+            branch_name = version_match.group(2)
+            return cadet_version, branch_name
+        else:
+            raise ValueError("Unable to determine CADET version from the output")
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(f"Command execution failed: {e}")
+
+
+def run_simulation_for_all_units(cadet_install_path: str = None) -> None:
```

Comment:
This function, as well as `get_all_unit_types` should be migrated to the actual test method(s)